task/bootc-rechunk/0.1/README.md

```diff
@@ -0,0 +1,45 @@
+# Bootc-Rechunk task
+
+## Overview
+
+The `bootc-rechunk` task is a Tekton task used to chunk an OCI image into smaller chunks for more efficient management, with the goal of reducing image size. This task is designed to work in conjunction with other container build tasks (e.g., `buildah-oci-ta`). It takes an existing image, processes it, and outputs a chunked OCI archive.
+
+## Inputs
+
+- **IMAGE**: The reference of the image to be chunked (required). This image will be used as the input for the rechunking process.
+  
+## Outputs
+
+- **OCI Archive**: A chunked OCI image that is output after the process completes. The output is saved to the specified path in the task parameters.
```

Comment:
There's several different output formats here; an OCI archive (aka `oci-archive:` in [containers-transport terms](https://github.com/containers/image/blob/main/docs/containers-transports.5.md)) is just one.

I think it's generally better to output `oci:` personally.

Although this relates to the above - if we pull from a registry it'd probably make sense to push to a registry?